content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -45,6 +45,8 @@ In this case, the virtual services will route all traffic to `v1` of each micros
     Because configuration propagation is eventually consistent, wait a few seconds
     for the virtual services to take effect.
 
+    __NOTE__: If you haven't already applied destination rules, follow the [Apply default destination rules](/docs/examples/bookinfo/#apply-default-destination-rules).
```

Comment:
add "instructions in" after "follow the" and make link title all caps: 
... follow the instructions in [Apply Default Destination Rules](/docs/...

Also please move the note up, before the first instruction.